.travis.yml

```diff
@@ -2,7 +2,7 @@ language: java
 sudo: false # faster builds
 
 jdk:
-- oraclejdk8
```

Comment:
@HuangXiaoyu3331 can you please rebase this PR from master? This change is not needed anymore

Reply:
I haved rebase it, please confirm whether it is correct